src-electron/generator/helper-zcl.js

```diff
@@ -2693,6 +2698,261 @@ function if_compare(leftValue, rightValue, options) {
   }
 }
 
+/**
+ *
+ * @param {*} type
+ * @param {*} dataType
+ * @param {*} packageIds
+ * @param {*} context
+ * @returns The size and sign of a zcl data type
+ */
+async function zcl_data_type_size_and_sign(
+  type,
+  dataType,
+  packageIds,
+  context
+) {
+  let result = 0
+  let isSigned = false
+  if (dataType.discriminatorName.toLowerCase() == dbEnum.zclType.bitmap) {
+    let bitmap = await queryZcl.selectBitmapByName(
+      context.global.db,
+      packageIds,
+      dataType.name
+    )
+    result = bitmap.size
+  } else if (dataType.discriminatorName.toLowerCase() == dbEnum.zclType.enum) {
+    let en = await queryZcl.selectEnumByName(
```

Comment:
How does this work if the same enum name is used in multiple clusters?

Reply:
Resolved this